### PR TITLE
ROX-11222: OSCI collector integration tests

### PR DESCRIPTION
**Pull request by Stringy**
## Description

A detailed explanation of the changes in your PR.

Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Updated documentation accordingly

**Automated testing**
  - [ ] Added unit tests
  - [ ] Added integration tests
  - [ ] Added regression tests

If any of these don't apply, please comment below.

## Testing Performed

TODO(replace-me)
Use this space to explain how you tested your PR, or, if you didn't test it, why you did not do so. (Valid reasons include "CI is sufficient" or "No testable changes")
In addition to reviewing your code, reviewers **must** also review your testing instructions, and make sure they are sufficient.

For more details, ref the [Confluence page](https://stack-rox.atlassian.net/wiki/spaces/StackRox/pages/855998488/Proposal+Explicitly+List+Testing+Steps+on+PRs) about this section.

### CODE REVIEW

File: .openshift-ci/jobs/integration-tests/run-integration-tests.sh [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        return 1
+    fi
+    return 0
+}
+
+function integration_tests_with_measurements() {
+    COLLECTOR_BCC_COMMAND="/tools/do_syscall_64.py -o /tmp/baseline.json" _ make -C "${SOURCE_ROOT}" integration-tests-baseline
+    COLLECTOR_BCC_COMMAND="/tools/do_syscall_64.py -o /tmp/benchmark.json" _ make -C "${SOURCE_ROOT}" integration-tests-benchmark
+
+    _ make -C "${SOURCE_ROOT}" ci-integration-tests
```
Comment (JoukoVirtanen) [inline .openshift-ci/jobs/integration-tests/run-integration-tests.sh:26]: What is the purpose of these underscores?

File: .openshift-ci/jobs/integration-tests/envbuilder.sh [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+    local GCP_SSH_KEY_FILE="$1"
+    shift
+
+    local retryCount=6
+    for _ in $(seq 1 $retryCount); do
+        _ gcloud compute ssh --strict-host-key-checking=no --ssh-flag="-o PasswordAuthentication=no" --ssh-key-file="${GCP_SSH_KEY_FILE}" "${GCP_VM_USER}@${GCP_VM_NAME}" --command "whoami" \
+            && exitCode=0 && break || exitCode=$? && sleep 15
+    done
+
+    instance_id="$(_ gcloud compute instances describe "${GCP_VM_NAME}" --format='get(id)')"
```
Comment (robbycochran) [inline .openshift-ci/jobs/integration-tests/envbuilder.sh:201]: Remove the usage of `_` here. It echos the command and the `instance_id` var will be invalid.

File: integration-tests/executor.go [hunk 3]
```diff
@@ -11,7 +11,7 @@ import (
 )
 
 var (
-	debug = false
+	debug             = false
```
Comment (ovalenti) [inline integration-tests/executor.go:14]: I wonder what is the reason for this formatting ?

**Review by ovalenti — COMMENTED** [review]
There are lots of places where ${GCP_VM_USER}@${GCP_VM_NAME} is written without braces or mixed. This is probably not very important.

**Review by ovalenti — APPROVED** [review]
Updating user names and accounts is planned as a follow-up.

LGTM

**Pull request merged**